aclk/agent_cloud_link.c

```diff
@@ -153,10 +153,6 @@ static int create_private_key()
         error("Claimed agent cannot establish ACLK - private key not found '%s' failed.", filename);
         return 1;
     }
-    if (unlikely(statbuf.st_size == 0)) {
```

Comment:
I think this check should stay, maybe with something like `statbuf.st_size < MIN_SIZE_SOMETHING`

Reply:
Why?

Reply:
Because a 1 byte file is not valid, is it?

Reply:
The validity check is loading the key into openssl. There is no definite value for MIN_SIZE_SOMETHING.